org.eclipse.scout.rt.platform.test/src/test/java/org/eclipse/scout/rt/platform/config/PropertiesHelperTest.java

```diff
@@ -899,36 +784,126 @@ else if ("dbSystemProperties".equals(configUrl)) {
     }
   }
 
-  /**
-   * Used in {@link PropertiesHelperTest} as a test implementation for {@link IJsonPropertyReader}.
-   */
-  @IgnoreBean
-  public static class TestJsonPropertyReader implements IJsonPropertyReader {
+  @Test
+  public void testParseJsonJavadocExample() {
+    PropertiesHelper propertiesHelper = new PropertiesHelper(mock(IPropertyProvider.class));
+    Map<String, String> decodedValue = propertiesHelper.parseJson("{\n"
+        + "  \"key1\": \"value1\",\n"
+        + "  \"key2\": null,\n"
+        + "  \"key3\": \"\",\n"
+        + "  \"key4\": 1,\n"
+        + "  \"key5\": \"1\",\n"
+        + "  \"key6\": true,\n"
+        + "  \"key7\": {\"key71\": \"value71\", \"key72\": \"value72\", \"key73\": \"value73\"},\n"
+        + "  \"key8\": [\"value81\", \"value82\", \"value83\"]\n"
+        + "}");
 
-    private final Map<String, Map<String, String>> m_propertyValueToMap = new HashMap<>();
+    HashMap<String, String> expectedMap = new HashMap<>();
+    expectedMap.put("key1", "value1");
+    expectedMap.put("key2", null);
+    expectedMap.put("key3", "");
+    expectedMap.put("key4", "1");
+    expectedMap.put("key5", "1");
+    expectedMap.put("key6", "true");
+    expectedMap.put("key7", "{\"key71\":\"value71\",\"key72\":\"value72\",\"key73\":\"value73\"}"); // changed?
```

Comment:
Add reason why not completly identical, e.g. JSON as a value might be reformatted, thus spaces might change...
Add this reason to on the JavaDoc of the parse method